packages/iris-grid/src/IrisGridTableModelTemplate.ts

```diff
@@ -1091,7 +1150,19 @@ class IrisGridTableModelTemplate<
 
     log.debug2('copyTotalsData', dataMap);
 
+    const dataBarColumnIndex = 2;
```

Comment:
this was the issue I was trying to describe earlier in the meeting. In the table that I created there were three columns (Name, MathGrade, GPA). If I want to access min or max from the map, I have to use the column index as the key to access it. I hard coded it to 2 since I know GPA will be the 3rd column in the example I generated but I need a way to dynamically find this